### PR TITLE
Added censorship to warning

### PR DESCRIPTION
**Pull request by davidecotten**

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -2046,13 +2046,16 @@ def create_quote_image(id, type, quote, name):
 
     #Is someone being profane?
     if message.channel.name != "serious_business":
-        content = message.content.lower().replace(' ', '')
+        content = message.content.lower()
         for profane_word in profane_words:
             if profane_word in content:
-                if random.randint(1, 50) == 1:
-                    msg = "Hey {member}! Shame on you. This is a christian server.\nSister <@194703127868473344> is always watching.".format(member=member.mention)
+                if random.randint(1, 1) == 1:
```
Comment (aceat64) [inline bot.py]: lol why? this means it'll always run

File: bot.py [hunk 2]
```diff
@@ -2046,13 +2046,16 @@ def create_quote_image(id, type, quote, name):
...
+                    for profane_word in profane_words:
+                        content = content.replace(profane_word, '||' + profane_word + '||')
+                    content = '. '.join(sentence.capitalize() for sentence in content.split('. '))
+                    msg = "Hey {member}! Shame on you. This is a christian server.\nSister <@194703127868473344> is always watching. I have fixed your message for you.\n\n {member} said: \"{content}\"".format(member=member.mention, content=content)
                     await client.send_message(message.channel, msg)
-
+                    await client.delete_message(message)
```
Comment (aceat64) [inline bot.py:2064]: Not sure I like the idea of deleting people's messages.

**Review by aceat64 — CHANGES_REQUESTED** [review]

File: bot.py (hunk 1 above)
Comment (davidecotten) [reply bot.py]: Oversight. I changed it for testing and forgot to revert.

File: bot.py (hunk 2 above)
Comment (davidecotten) [reply bot.py:2064]: Their message must be deleted for the censorship to make sense. The message gets censored and reposted anyway.

**Review by aceat64 — APPROVED** [review]

**Pull request merged**